### DISCUSSION TITLE
Maintenance status placeholders for Bungee

### DISCUSSION THREAD

**Discussion by tlgnkl** [Zrips/CMI#5785] [Feature Requests]
The plugin has a server maintenance mode and there are placeholders for it
![image](https://user-images.githubusercontent.com/67519432/123541154-a672ea00-d74b-11eb-978a-b1f0db0db115.png)
But there are no placeholders for Bungee servers in the format %cmi_maintenance_state_[serverName]%
If they existed, then CMI could replace another plugin - https://www.spigotmc.org/resources/maintenance-mode.2548/

**Comment by Gyztor** [comment]
I feel there should be cross server of the maintenance placeholder so you can easily show if a server is in maintenance or not with holograms

**Reply by MaHl111** [reply]
Well, there should also be dedicated name/symbol for maintenance on/off state as there is for AFK. maintenance_state will return True/False, while maintenance_message returns locale maintenance message that's shows when players can't the access server, none of which is practically feasible for most common use-cases. Would be nice to have both of these features so you can see server as **Online** or **in Maintenance** instead of having to check for each state and do additional logic

**Reply by Gyztor** [reply]
Yeah i can see that being a good idea too since you can make server holograms in front of server change portals that say have like
```
Server is: {Online|Offline|In Maintenance}
{players online in server} / {player slots}
<appears only when in maintenance for specified server>Reason for maintenance: {insert reason}
```
and do this all for a different server not the one you are currently on as right now maintenance placeholders only work for the server it is currently used on. and having states would allow for the first line to actually work.

**Reply by MaHl111** [reply]
Exactly, placeholders currently are being parsed only when player is online on server, which means value won't show up outside of the server (being it server list or supposedly bungee) because player is considered **not being on that specific server**. Will see how this will be handled going forward. **Bump it tho**

**Comment by Gyztor** [comment]
Bump for bringing a useful feature.
